harness/determined/common/schemas/expconf/_gen.py

```diff
@@ -2675,6 +2675,26 @@
                 "null"
             ],
             "default": null
+        },
```

Comment:
why do we ever need to differentiate these? sorry coming back to this after not looking at it for a while.

Reply:
Honoring `nil` values is equivalent to knowing if the user provided a value or not.  Trampling them would mean that you couldn't tell if you were looking at a defaulted value of if the user provided e.g. an empty list explicitly.

Reply:
Makes sense.